starter-code/scripts/article.js

```diff
@@ -6,16 +6,25 @@ function Article (opts) {
   // TODO: Use the JS object passed in to complete this constructor function:
   // Save ALL the properties of `opts` into `this`
   // lots of this.someProperty = opts.property
+  this.title = opts.title;
+  this.category = opts.category;
+  this.author = opts.author;
+  this.authorUrl = opts.authorUrl;
+  this.publishedOn = opts.publishedOn;
+  this.body = opts.body;
 }
 
 Article.prototype.toHtml = function() {
   var $newArticle = $('article.template').clone();
+
   /* TODO: This cloned article is no longer a template,
   as it now has real data attached to it! We need to account
   for that before this current article gets rendered to our
   DOM. */
   // one line of code to remove the class that hides the template so that the articles will actually show
 
+  $newArticle.removeClass('template');
```

Comment:
Nice! Also note that you can  add the .removeClass('template') , the end of line 18, to make var $newArticle = $('article.template').clone().removeClass('template) 
THE POWER OF CHAINING!!!